fusor-ember-cli/app/templates/edit-global-service-config.hbs

```diff
@@ -25,6 +25,8 @@
                               {{text-f label=planParameter.displayId value=planParameter.value type=planParameter.inputType labelSize='col-md-6' inputSize='col-md-5' isRequired=false helpText=planParameter.description}}
                             {{/if}}
                           {{/if}}
+                        {{else}}
+                          There are no global parameters to edit.
```

Comment:
If `===` is correct, then at least we have a message to the user, rather than a blank form.